### ISSUE TITLE
Uncommented Gen 4+ abilities in pokemon_expansion

### ISSUE THREAD

**Issue by ExpoSeed** [rh-hideout/pokeemerald-expansion#10]
Some mons in `base_stats.h` have Gen 4+ abilities assigned to them instead of them being commented out. Example:
```c
    [SPECIES_CAMERUPT] =
    {
...
        .abilities = {ABILITY_MAGMA_ARMOR, ABILITY_SOLID_ROCK},
...
    },
```
Solid Rock was introduced in Gen 4. If this ability gets loaded in game, it displays as garbage and does nothing.

All abilities like this should be revised to the following:
```c
    [SPECIES_CAMERUPT] =
    {
...
        .abilities = {ABILITY_MAGMA_ARMOR, ABILITY_NONE}, //ABILITY_SOLID_ROCK},
...
    },
```
So that one could delete all instances of `ABILITY_NONE}, //` to automatically get Gen 4+ abilities for battle_engine.

**Comment by Gamer2020** [comment]
Will work on this.

**Comment by Gamer2020** [comment]
How should the hidden abilities be handled in regards to this?

**Comment by ExpoSeed** [comment]
`.abilityHidden = ABILITY_NONE, // ABILITY_XXXXXX` for now, we can look into doing these with `ifdef`s later once all these abilities have been marked and fixed properly

**Comment by Gamer2020** [comment]
One last question. What if both abilities are not in gen3?

**Comment by ultima-soul** [comment]
Fixed in #29
